makefile

```diff
@@ -1,33 +1,35 @@
 SHELL=/bin/bash
 
 # i.e. fedora28.yaml
-ALL_TEMPLATES=$(wildcard templates/*.yaml)
+ALL_META_TEMPLATES=$(wildcard templates/templates/*.yaml)
```

Comment:
Could meta templates be in `tempaltes/meta`?

Reply:
No, ansible would not find it there. But I removed the nesting in favor of dist/ and templates/